### PR TITLE
Poly "<b>" Speech fix

### PR DESCRIPTION
**Pull request by EmeraldCandy**
## What Does This PR Do
Fixes: #25257
Fixed polly trying to add command boldening `<b>` to their message if they hear a command member speak

## Why It's Good For The Game
bugfix good

## Testing
image

## Changelog
:cl:
fix: Poly Command Bold Mimic Fix
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 1]
```diff
@@ -697,6 +697,8 @@
 /mob/living/simple_animal/parrot/proc/parrot_hear(message)
 	if(!message || stat)
 		return
+	if(findtext(message,"<b>",1,0))
+		message = copytext(message,4,(length(message)-3))
```
Comment (Burzah) [inline code/modules/mob/living/simple_animal/parrot.dm:701]:
```suggestion
	if(findtext(message, "<b>", 1, 0))
		message = copytext(message, 4, (length(message) - 3))
```

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 2]
```diff
@@ -697,6 +697,8 @@
 /mob/living/simple_animal/parrot/proc/parrot_hear(message)
 	if(!message || stat)
 		return
+	if(findtext(message, "<b>", 1, 0))
+		message = copytext(message, 4, (length(message) - 3))
```
Comment (Contrabang) [inline code/modules/mob/living/simple_animal/parrot.dm:701]: Use `strip_html_tags` here instead

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/parrot.dm (hunk 2 above)
Comment (EmeraldCandy) [reply code/modules/mob/living/simple_animal/parrot.dm]: Didnt know that command existed! done

**Review by Spaghetti-bit — APPROVED** [review]

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 3]
```diff
@@ -697,6 +697,7 @@
 /mob/living/simple_animal/parrot/proc/parrot_hear(message)
 	if(!message || stat)
 		return
+	message = strip_html_tags(message)
 	speech_buffer.Add(message)
```
Comment (Contrabang) [inline code/modules/mob/living/simple_animal/parrot.dm:701]: Speed 😎 
```suggestion
	speech_buffer.Add(strip_html_tags(message))
```

Comment (EmeraldCandy) [reply code/modules/mob/living/simple_animal/parrot.dm:701]: that probably a way better way to do it, yeah

**Review by Burzah — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

**Review by Contrabang — APPROVED** [review]
seeeend it

**Pull request merged**